cli/command/system/version.go

```diff
@@ -125,6 +141,11 @@ func runVersion(dockerCli command.Cli, opts *versionOptions) error {
 			Status: "Template parsing error: " + err.Error()}
 	}
 
+	kubeVersion, err := getKubernetesVersion(dockerCli, cmd)
+	if err != nil {
+		return err
```

Comment:
This could cause `version` to exit with an error if the stack API isn't available, which would result in the user not seeing any version info.

I don't think that's the correct behaviour.  I think the error case should set any any empty fields in `kubeVersion`  to "unknown", print the error as a warning, and continue with printing the version. For example maybe `getKubernetesVersion` is able to get a kube version, but not the stack API version